Added runningTasks to BasicQueryStats

hitarth opened the pull request:
Add runningTasks to BasicQueryStats and added cluster level aggregated runningTasks to v1/cluster. This would help in monitoring total number of running tasks on the cluster. 

```
== RELEASE NOTES ==
General Changes
* Add support for running task count ``runningTasks`` at the cluster level via ``v1/cluster`` endpoint
```

hitarth commented:
@swapsmagic, @tdcmeehan Can you please help review.

swapsmagic commented:
Can you add a unit test to verify the endpoint behavior with runningTasks? Looks like we are missing unit tests around the ClusterStatsResource, so this will be a good start.

hitarth commented:
> Can you add a unit test to verify the endpoint behavior with runningTasks? Looks like we are missing unit tests around the ClusterStatsResource, so this will be a good start.

Thanks for taking a look. I have added a unit test.

swapsmagic commented on presto-tests/src/test/java/com/facebook/presto/server/TestClusterStatsResource.java:
```diff
@@ -0,0 +1,98 @@
+        client = null;
+    }
+
+    @Test(timeOut = 60_000, enabled = false)
```
why is the test marked as disabled?

hitarth replied:
Enabled the test.

swapsmagic commented on presto-tests/src/test/java/com/facebook/presto/server/TestClusterStatsResource.java:
```diff
@@ -0,0 +1,98 @@
+        ClusterStatsResource.ClusterStats clusterStats = client.execute(request, createJsonResponseHandler(jsonCodec(ClusterStatsResource.ClusterStats.class)));
+
+        assertNotNull(clusterStats);
+        assertTrue(clusterStats.getActiveWorkers() > 0);
```
it should be a fixed value, i think in this case it will be 4 workers + 1 coordinator so a total of 5.

hitarth replied:
Updated the test

swapsmagic reviewed (changes requested):
Can you add a release note as well given we are exposing more info via the endpoint, we should put that in the release note. It might impact clients consuming this endpoint.

hitarth commented:
> Can you add a release note as well given we are exposing more info via the endpoint, we should put that in the release note. It might impact clients consuming this endpoint.

Updated release notes.

swapsmagic reviewed (approved):
One more unit test around DistributedClusterStatsResource and it's good.

swapsmagic commented:
Release notes should be something like this: We have a guideline for that: https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines

* Add support for running task count ``runningTasks`` at the cluster level via ``v1/cluster`` endpoint

hitarth commented:
Thanks for reviewing. 

> One more unit test around DistributedClusterStatsResource and it's good.

I have updated an existing test in ``DistributedClusterStatsResource``

> Release notes should be something like this: We have a guideline for that: https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines
> 
> * Add support for running task count `runningTasks` at the cluster level via `v1/cluster` endpoint

Updated release notes as per guidelines.
